.ci/resolve-kata-dependencies.sh

```diff
@@ -62,13 +62,14 @@ clone_repos() {
 		echo "Cloning ${repo}"
 		go get -d "${repo}" || true
 		repo_dir="${GOPATH}/src/${repo}"
+		pushd "${repo_dir}"
 
-		# The tests repository is cloned and checkout to the PR branch directly in
-		# the CI configuration (e.g. jenkins file or zuul config), because we want
-		# to have latest changes of this repository, since the job starts. So we
-		# need to verify if we are already in the PR branch, before trying to
-		# fetch the same branch.
-		if [ ${repo} == ${tests_repo} ]
+		# When we have a change from the tests repo, the tests repository is cloned
+		# and checkout to the PR branch directly in the CI configuration (e.g. jenkins
+		# config file or zuul config), because we want to have latest changes
+		# of this repository since the job starts. So we need to verify if we
+		# are already in the PR branch, before trying to fetch the same branch.
+		if [ ${repo} == ${tests_repo} ] && [ "${repo}" == "${kata_repo}" ]
```

Comment:
nit - whilst you are there, just below here, there looks like an indent syntax error - this line:

`current_branch=$(git rev-parse --abbrev-ref HEAD)`

needs indenting?

Reply:
:-) I pulled the source, and that line looks to be one tab short of an indent in my editor. Odd, as I don't see any space/tab mixing in the file either.